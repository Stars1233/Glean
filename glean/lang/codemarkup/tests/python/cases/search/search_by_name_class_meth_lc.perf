{
  "@generated": null,
  "facts_searched": {
    "python.ContainingTopLevelDeclaration.4": 1,
    "python.DeclarationDefinition.4": 1,
    "python.DeclarationLocation.4": 1,
    "python.LocalNameLowerCase.4": 1,
    "python.SearchByLocalName.4": 1
  },
  "full_scans": [ ]
}