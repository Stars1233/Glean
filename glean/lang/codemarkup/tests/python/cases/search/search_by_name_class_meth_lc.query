query: codemarkup.search.SearchByName { name = "meth", language = Python, searchcase = Insensitive }
perf: true
transform: [normord, []]
