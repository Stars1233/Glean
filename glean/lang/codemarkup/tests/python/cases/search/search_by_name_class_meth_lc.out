[
  "@generated",
  {
    "key": {
      "entity": {
        "python": {
          "decl": {
            "func": { "key": { "name": { "key": "some.foo.Foo.meth" } } }
          }
        }
      },
      "kind": 7,
      "language": 4,
      "location": {
        "file": { "key": "some/foo.py" },
        "location": { "span": { "length": 42, "start": 260 } },
        "name": "some.foo.Foo.meth"
      },
      "name": "meth",
      "searchcase": 1
    }
  }
]